devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -77,7 +78,12 @@ def parse(String description) {
             }
         }
         else if (description?.startsWith('zone status')) {
-            event = parseIasButtonMessage(description)
+            if((device.getDataValue("model") == "SOS-EM")){
```

Comment:
What happens if you don't include this change to `parse`? Does `parseIasButtonMessage` not do the right thing for this device?

Reply:
I checked the log, the device only report release event, no press event, so parseIasButtonMessage can't return a pushed event for it. this why i not let it be handled by parseIasButtonMessage

Reply:
What is the `zone status` message when the button is pressed? The issue is we cannot make a change like this to `parse` because this DTH runs locally on most hubs so this code will not be executed.

Reply:
hm... i see description is zone status 0x0001 -- extended status 0x00 when press, but no log occurr when release

Reply:
Unfortunately that means this DTH won't work for this device. However I think you should be able to use the `SmartSense Button` DTH instead and won't need these changes to `parse` (you will still need the changes to `configure`).